dart/spry/pubspec.yaml

```diff
@@ -0,0 +1,7 @@
+name: spry_app
+
+environment:
+  sdk: '>=3.2.0 <3.3.0'
+
+dependencies:
+  spry: ^3.2.0
```

Comment:
not remember exactly, but does it guarantee to be on 3.2.x?

Reply:
Yes, it is guaranteed to always work on 3.2.x